app/renderer/components/main/braveryPanel.js

```diff
@@ -993,6 +994,7 @@ const styles = StyleSheet.create({
   braveryPanel__body__advanced__control: {
     display: 'grid',
     gridColumnGap: '1rem',
+    gridTemplateColumns: 'max-content max-content',
```

Comment:
This prevents the strings inside the dropdowns and the labels from wrapping, without forcing the same width for each column.